### ISSUE TITLE
Allow Popeye to be used by more than one user on a host (/tmp/popeye.log permission problem)

### ISSUE THREAD

**Issue by jim-barber-he** [derailed/popeye#301]
<img src="https://raw.githubusercontent.com/derailed/popeye/master/assets/popeye_feature.png" align="right" width="100" height="auto"/>

<br/>
<br/>
<br/>

**Is your feature request related to a problem? Please describe.**

I manage multiple clusters, and on my system I sudo to different user IDs to manage them.
If I run popeye as one user, it creates a `/tmp/popeye.log` owned by the user.
If I then try to run popeye as another user it errors like so:
```
$ popeye version     
Unable to create Popeye log file open /tmp/popeye.log: permission denied. Exiting...
```

It would be good to be able to have popeye work around this limitation.

**Describe the solution you'd like**

To be able to run popeye as one user, then be able to run it as another user without the original user (or root) having to remove the /tmp/popeye.log file first.

**Describe alternatives you've considered**

As for implementation there are so many valid ways you can tackle this, that it would be up you.
E.g.
- Create a /tmp/popeye-$UID.log file instead.
- Create a /tmp/popeye-$PID.log file instead.
- Keep creating /tmp/popeye.log and if it can't be written to fall back on one of the options above.
- Keep creating /tmp/popeye.log and if it can't be written to proceed with a warning (configurable to be turned off) and don't produce a log file at all.
- Provide an option to not create a popeye.log file at all (configurable via an environment variable would be nice)
- Create a /tmp/$UID/popeye.log instead.
And so on.

**Additional context**

Hopefully the above captures my request.

**Comment by derailed** [comment]
@jim-barber-he Thank you for the heads up! 
In the next drop, I have added a cli arg to disable logs aka `--logs none`. 
This won't automatically create a log file and use stdout instead.
Another thing you will be able to do is use  `--logs /tmp/$USER/popeye.log` so you can track logs per users.
Would this help?

**Comment by derailed** [comment]
Fixed v0.21.6

**Comment by jim-barber-he** [comment]
> @jim-barber-he Thank you for the heads up! In the next drop, I have added a cli arg to disable logs aka `--logs none`. This won't automatically create a log file and use stdout instead. Another thing you will be able to do is use `--logs /tmp/$USER/popeye.log` so you can track logs per users. Would this help?

Sounds perfect. Thanks.
